modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -1209,2824 +1208,1603 @@ class SampleCodeTest {
-    }
-}
-// end::local-win-conflict-resolver[]
-
-// tag::remote-win-conflict-resolver[]
-class RemoteWinConflictResolver: ConflictResolverProtocol {
-    func resolve(conflict: Conflict) -> Document? {
-        return conflict.remoteDocument
+    
+    // MARK: -- URLEndpointListener
+    func dontTestTLSIdentityCreate() throws {
+        // tag::p2psync-act-tlsid-create[]
+        try TLSIdentity.deleteIdentity(withLabel: "couchbaselite-server-cert-label");
+        let attrs = [certAttrCommonName: "CBL-Server"]
+        let identity = try TLSIdentity.createIdentity(forServer: true,
+                                                      attributes: attrs,
+                                                      expiration: nil,
+                                                      label: "couchbaselite-server-cert-label")
+        // end::p2psync-act-tlsid-create[]
+        print(identity)
+    }
+    
+    func dontTestDeleteIdentity() throws {
+        // tag::p2psync-act-tlsid-delete[]
+        try TLSIdentity.deleteIdentity(withLabel: "couchbaselite-client-cert-label")
+        // end::p2psync-act-tlsid-delete[]
+    }
+    
+    func dontTestImportTLSIdentity() throws {
+        // tag::p2psync-act-tlsid-import[]
+        
+        let path = Bundle.main.path(forResource: "identity/client", ofType: "p12")
+        let clientCertData = try NSData(contentsOfFile: path!, options: []) as Data
+        let identity = try TLSIdentity.importIdentity(withData: clientCertData,
+                                                      password: "123",
+                                                      label: "couchbaselite-client-cert-label")
+        // end::p2psync-act-tlsid-import[]
+        print(identity)
+    }
+    
+    // MARK: -- QUERY RESULT SET HANDLING EXAMPLES
+    
+    func donTestQuerySyntaxAll() throws {
+        // tag::query-syntax-all[]
+        let database = try Database(name: "hotel")
+        let query = QueryBuilder.select(SelectResult.all()).from(DataSource.database(database))
+        
+        // end::query-syntax-all[]
+        
+        print(query)
     }
-}
-// end::remote-win-conflict-resolver[]
-
-// tag::merge-conflict-resolver[]
-class MergeConflictResolver: ConflictResolverProtocol {
-    func resolve(conflict: Conflict) -> Document? {
-        let localDict = conflict.localDocument!.toDictionary()
-        let remoteDict = conflict.remoteDocument!.toDictionary()
-        let result = localDict.merging(remoteDict) { (current, new) -> Any in
-            return current // return current value in case of duplicate keys
+    
+    func dontTestQueryAccessAll() throws {
+        let query = QueryBuilder.select(SelectResult.all()).from(DataSource.database(database))
+        var hotels = [String: Any]()
+        
+        // tag::query-access-all[]
+        let results = try query.execute()
+        for row in results {
+            let docsProps = row.dictionary(at: 0)?.toDictionary() // <.>
```

Comment:
As each docsProps is using ! so I think it is better to do`row.dictionary(at: 0)!`

Reply:
Also there is no need to convert from DictionaryObject to Dictionary.